### ISSUE TITLE
improve the dark theme

### ISSUE THREAD

**Issue by yyoulal** [sublimehq/sublime_merge#353]
**Problem description**

I have bought the license and I am glad I did ; it is really a great piece of software.
However, I find the diffs in the dark theme not very legible. The color scheme is beautiful but the low contrast of the diff insertions/deletions make them somehow hard to read.

**Preferred solution**

Review the diff colours of the current dark theme, as I believe it should look perfect out of the box.
Make available more color schemes, including the darker ones we have in ST3.

**Comment by wbond** [comment]
This is a fairly subjective item.

Since you are a paid user, you have full access to customize the theme and color scheme. In this case, you'll want to look at https://www.sublimetext.com/docs/3/dev/color_schemes.html#customization and then create a customization of `Marana.sublime-color-scheme`. 

Here are the default rules that affect the diff background (and foreground tweaks):

```json
{
   "rules":
    [
        {
            "scope": "diff.deleted",
            "background": "hsla(357, 45%, 60%, 0.15)",
            "foreground_adjust": "l(+ 5%)"
        },
        {
            "scope": "diff.deleted.char",
            "background": "hsla(357, 60%, 60%, 0.30)",
            "foreground_adjust": "l(+ 10%)"
        },
        {
            "scope": "diff.inserted",
            "background": "hsla(180, 45%, 60%, 0.15)",
            "foreground_adjust": "l(+ 5%)"
        },
        {
            "scope": "diff.inserted.char",
            "background": "hsla(180, 60%, 60%, 0.30)",
            "foreground_adjust": "l(+ 10%)"
        },
        {
            "scope": "diff.deleted.side-by-side",
            "background": "hsl(210, 15%, 27%)",
            "foreground_adjust": "l(+ 5%)"
        },
        {
            "scope": "diff.inserted.side-by-side",
            "background": "hsl(210, 15%, 27%)",
            "foreground_adjust": "l(+ 5%)"
        },
    ]
}
```

**Comment by jdgregson** [comment]
@wbond Will, is there any way to get the entire dark theme in a `.sublime-theme` and `.sublime-color-scheme` format? I'm glad that Sublime Merge is themeable beyond the two built-in themes, but I don't have the time to create an entire theme from scratch and have not found any themes for Sublime Merge on the internet. It would be much easier if I could take what's already done in the dark theme and make the minor changes I'd like to see for my own use.

**Comment by jdgregson** [comment]
@wbond Never mind, I was able to extract them myself using PackageResourceViewer.

"Merge Dark.sublime-theme":  https://pastebin.com/keNt6AbS 
"Merge.sublime-theme":  https://pastebin.com/NCQjJaMb

**Comment by wbond** [comment]
@jdgregson If you just want to tweak colors for Merge Dark, I highly recommend creating a theme customization (https://www.sublimetext.com/docs/3/themes.html#customization) and just changing the various variables that control color.

**Comment by wbond** [comment]
That, or you can `extend` Merge Dark

**Comment by JosephTLyons** [comment]
I know that the themes are customizable, but I’m just going to weigh in and say that I agree with OP, that the default diff colors in dark mode are not that great for me either.  I’ll tweak the theme, but I figured the feedback would be good, to show that OP isn’t the only one.

**Comment by JosephTLyons** [comment]
I don't own Sublime Text, Sublime Merge is my first product of yours.  I'm having a bit of trouble figuring this out.  I've placed a file in this directory:

`/Users/josephlyons/Library/Application Support/Sublime Merge/Packages/User`

With the file name being: `Dark.sublime-theme`

I've used the template you provided [here](https://github.com/sublimehq/sublime_merge/issues/353#issuecomment-471008566) and tweaked some values, but nothing is occurring in Merge then I exit out and reopen the app.  Is that the correct procedure to get the app to read the new settings?

Maybe my file name is incorrect?
